src/Germline.cpp

```diff
@@ -88,191 +91,25 @@ Germline::Germline(YAML::Node root) {
       }
     }
 
+    // Parse germline emission data.
     std::tie(state_names, probs) =
         ParseStringProbMap(gstate["emissions"]["probs"]);
     assert(IsEqualStringVecs(state_names, alphabet));
 
     for (unsigned int j = 0; j < state_names.size(); j++) {
       emission_matrix_(alphabet_map_[state_names[j]], gindex) = probs[j];
     }
+
+    // Parse germline bases and rates.
+    bases_[gindex] =
+        alphabet_map_[gstate["extras"]["germline"].as<std::string>()];
+    /// @todo HOW SHOULD I POPULATE THIS??? - DELETE ME!
```

Comment:
Yeah, as described I think that we should just have v1 have the standard 4 Gamma mixture of rates. So no per-site rates at this point. If you agree, update comment as such?